tensorflow/core/common_runtime/dml/dml_execution_context.cc

```diff
@@ -211,6 +211,10 @@ D3D12_COMMAND_LIST_TYPE DmlExecutionContext::GetCommandListTypeForQueue()
     std::shared_ptr<DmlCommandList> command_list,
     std::shared_ptr<DmlCommandQueue> command_queue, uint32_t batch_flush_size,
     uint32_t batch_flush_time_us) {
+#if _WIN32
+  SetThreadDescription(GetCurrentThread(), L"TFDML Execution Thread");
```

Comment:
Appears this is a dedicated thread owned by DML, and that no other thread will conflict or compete with naming.